pkg/kubelet/nodestatus/setters.go

```diff
@@ -612,6 +612,10 @@ func getCurrentRuntimeReadiness(runtimeCondition *v1.NodeCondition, workloadType
 	statusSet := false
 	for _, status := range runtimeServiceStatus {
 		if status == true {
+			if runtimeCondition.Status == v1.ConditionTrue {
+				// runtime was already ready; no change
+				return runtimeCondition
+			}
 			runtimeCondition.Status = v1.ConditionTrue
 			runtimeCondition.Reason = fmt.Sprintf("At least one %s runtime is ready", workloadType)
 			recordEventFunc(v1.EventTypeNormal, fmt.Sprintf("%s is ready", runtimeCondition.Type))
```

Comment:
My understanding is that those events are used as healthy check report, they would update revision # and last updated time. Skip all the events that are the same might have other issue such as node was not refreshed in certain time period. Perhaps record those event in a longer interval.

Do you know currently how often it records such event for a healthy node?